bin/elixir.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
Please rename to `bin/install-elixir.sh` to match the Erlang script.

Reply:
There is no `bin/install-erlang.sh` these are named:
- `apt-erlang.sh`
- `brew-erlang.sh`
- `yum-erlang.sh`
- `pkg-erlang.sh`

Hopwever I do agree that `bin/install-elixir.sh` is a better name.